src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -71,7 +71,11 @@ export default {
         .filter(({ denom }) => denom === this.denom)
         .reduce((sum, { amount }) => sum + Number(amount), 0)
         .toFixed(6)
-      return stakingRewards > 0 ? stakingRewards : this.claimedReward.amount
+      return stakingRewards > 0
+        ? stakingRewards
+        : this.claimedReward
+        ? this.claimedReward.amount
```

Comment:
indent

Reply:
<img width="380px" src="https://user-images.githubusercontent.com/40721795/76516850-7ae7c580-645c-11ea-89aa-fed1f843ec91.png">

Reply:
Does it revert to this formatting or does it keep the indentation on lint fix? I don't think this is good to read.

Reply:
I think this is the same indentation we had before no? I will just refactor

Reply:
don't worry. I think eslint is forcing this